core/shared/platform/windows/win_memmap.c

```diff
@@ -19,6 +19,9 @@ void * os_mmap(void *hint, size_t size, int prot, int flags)
         /* integer overflow */
         return NULL;
 
+    if (request_size == 0)
+        request_size = 1;
```

Comment:
mmap maps one page at least, passing 1 byte might cause unexpected behaviour, here had better change to:
request_size = page_size;